meson.build

```diff
@@ -28,6 +28,12 @@ gtk_layer_shell = dependency('gtk-layer-shell-0')
 
 pulse = dependency('libpulse')
 
+
+comp = meson.get_compiler('cpp')
+
+sass_lib = comp.find_library('libsass')
```

Comment:
I'll try to figure something out, the reason why I did it that way is because (at least on my distro) libsass did not ship with any cmake or other build files that meson would find.

ig I'll look into meson some more

Reply:
libsass should build and install a pkgbuild file (```libsass.pc```) by default, which can be used by meson. What is your distro, that it doesn't ship it?

Reply:
I'm using arch and the file is there. maybe just my mistake